app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+app.use(express.json())
+const { fetchAllTopics } = require('./controllers/controllers')
+const {handles404Errors, handles400Errors} = require('./handlesErrors')
+
+app.get("/api/topics", fetchAllTopics)
+
+
+
+
+app.use(handles404Errors)
```

Comment:
Would recommend moving the error handling for the 404 into the app.js, as it should only take two arguments so would not be considered middlewear. You will also need a path for a 500 (which can be dealt with in your error handling file), as opposed to 400 error.